strawberryfields/backends/gaussianbackend/backend.py

```diff
@@ -196,7 +196,7 @@ def measure_fock(self, modes, shots=1, select=None):
         if shots != 1:
             if select is not None:
```

Comment:
Looks ok now, but shouldn't this be placed outside of `if shots != 1:` since we don't support postselection generally (yet)? (also for `measure_threshold`)

Reply:
Also, is postselection something that we don't support _yet_ in the Gaussian backend, or will never support? If never, then should we not remove `select` as an argument in `measure_fock` and `measure_threshold`, and also remove the corresponding sentences in docstrings?

Reply:
The reason we need to keep the `select` argument is that this class inherits from `BaseBackend`, as does the Fock and TF backends. Therefore the method docstrings must match that of the base class.

Reply:
For Fock measurements I don't see how. For Threshold measurements it is possible in principle, but I am not there is any interest in doing it. A reason why select is there is just to keep consistency with the other backends, but not sure if it is worth or not keeping it. I suspect that should be dealt with in a different PR. This PR only tries to include threshold measurements at the same level as Fock measurements